trpcultivatetheme/css/layout/frontpage.css

```
@@ -130,8 +138,51 @@
 }
 
 /* Image placement in tiles */
+/* Bottom left half */
+.tripalcultivate-theme-tiles-banner-wide {
+  background-size: auto;
```

While still not quite getting us where we want responsive-wise using `contain` rather then `auto` here greatly improves the display (see review for screenshot comparison)

```suggestion
  background-size: contain;
```